example_configs/activemq.yml

```diff
@@ -0,0 +1,60 @@
+lowercaseOutputLabelNames: true
+blacklistObjectNames:
+  - "org.apache.activemq:clientId=*,*"
+whitelistObjectNames:
+  - "org.apache.activemq:destinationType=Queue,*"
+  - "org.apache.activemq:destinationType=Topic,*"
+  - "org.apache.activemq:type=Broker,brokerName=*"
+  - "org.apache.activemq:type=Topic,brokerName=*"
+
+rules:
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*), destinationType=Queue, destinationName=(\S*)><>(\w+)
+  name: activemq_queue_$3
+  attrNameSnakeCase: true
+  labels:
+    destination: $2
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*), destinationType=Topic, destinationName=(\S*)><>(\w+)
+  name: activemq_topic_$3
+  attrNameSnakeCase: true
+  labels:
+    destination: $2
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>CurrentConnectionsCount
+  name: activemq_connections
+  type: GAUGE
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>TotalConnectionsCount
+  name: activemq_connections_total
+  type: COUNTER
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>TotalConsumerCount
+  name: activemq_consumers_total
+  type: COUNTER
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>TotalDequeueCount
+  name: activemq_dequeue_total
+  type: COUNTER
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>TotalEnqueueCount
+  name: activemq_enqueue_total
+  type: COUNTER
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>TotalMessageCount
+  name: activemq_messages_total
+  type: COUNTER
+
+- pattern: org.apache.activemq<type=Broker, brokerName=(\S*)><>TempPercentUsage
```

Comment:
Is the raw data for this available?

Reply:
Yes I have from `jconsole`: 
![image](https://user-images.githubusercontent.com/691587/64175823-8b2a4280-ce32-11e9-9ffb-0d95ba24ce3c.png)

Reply:
That's a no then, there's only the percentage.